torchtitan/parallelisms/parallelize_llama.py

```diff
@@ -123,18 +123,10 @@ def get_tp_parallel_strategy(
 
     This function handles the special case of using float8 with tensor parallelism.
     """
-    if job_config.training.enable_fp8_linear:
-        from float8_experimental.float8_linear import Float8Linear, TensorScalingType
-
-        if any(
```

Comment:
removing since this is inspecting non-public information.  In a future PR I plan to add a torchtitan configuration for delayed scaling, and we can safely check that.

Reply:
which is non-public? `m.scaling_type_w` or `Float8Linear` ?

Reply:
`TensorScalingType` is public, but the fact that it's stored on a field of `Float8Linear` is not public and may change.

Reply:
`Float8Linear` is not public, it's an implementation detail